package.json

```diff
@@ -85,6 +87,7 @@
     "jest": "^24.1.0",
     "jest-fetch-mock": "^2.1.1",
     "jest-puppeteer": "^4.0.0",
+    "jsdom": "13.2.0",
```

Comment:
Should we remove the eslint-inline disable here now?: https://github.com/ProjectMirador/mirador/blob/master/setupJest.js#L3